src/adagrams.js

```diff
@@ -1,15 +1,206 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const oringinalPool = [
```

Comment:
It would be more efficient to have an object keeping track of how many of each letter is in the pool and then having the code create an array of all of the letters for you. This ensures there are no mistakes in making sure you have the correct number of each letter.